examples/extension/README.md

```diff
@@ -1,84 +1,64 @@
 ## Introduction
 
-This repository demonstrates how to create an extension archive to use with `otel.javaagent.experimental.extensions`
-configuration option of the OpenTelemetry Java instrumentation agent.
-
-For every extension point provided by OpenTelemetry Java instrumentation, this repository contains an example of
-its usage.
-
-Please carefully read both the source code and Gradle build script file `build.gradle`.
-They contain a lot of documentation and comments explaining the purpose of all major pieces.
-
-## How to use extension archive
-
-When you build this project by running `./gradlew build` you will get a jar file in
-`build/libs/opentelemetry-java-instrumentation-extension-demo-1.0-all.jar`.
-Copy this jar file to a machine running the application that you are monitoring with
-OpenTelemetry Java instrumentation agent.
-
-Assuming that your command line looks similar to this:
-```
-java -javaagent:path/to/opentelemetry-javaagent-all.jar \
-     -jar myapp.jar
-```
-change it to this:
-```
-java -javaagent:path/to/opentelemetry-javaagent-all.jar \
-     -Dotel.javaagent.experimental.extensions=path/to/extension.jar
-     -jar myapp.jar
-```
-specifying the full path and the correct name of your extensions jar.
-
-## Embedded extensions
-
-It is also possible to embedded you extension archive right inside OpenTelemetry Java Agent.
-This produces a single jar file thus simplifying deployment.
-Please consult `extendedAgent` task in [build.gradle](build.gradle) file for more information.
-When using Java Agent with embedded extension, the `-Dotel.javaagent.experimental.extensions`
-command line option is not needed anymore.
-Just `-javaagent` is sufficient.
+Extensions add new features and capabilities to the agent without having to create a separate distribution (for examples and ideas, see [Use cases for extensions](#use-cases-for-extensions)). 
 
+The contents in this folder demonstrate how to create an extension for the OpenTelemetry Java instrumentation agent, with examples for every extension point. 
+
+> Read both the source code and the Gradle build script, as they contain documentation that explains the purpose of all the major components.
+
+## Build and execute extension files
+
+To build this extension project, run `./gradlew build`. You can find the resulting jar file in `build/libs/`. 
+
+To execute the extension:
```

Comment:
```suggestion
To add the extension to the instrumentation agent:
```

You don't "execute" the extension. You add it to the agent which monitors your JVM/application